homeassistant/helpers/service.py

```diff
@@ -54,17 +49,18 @@
 from homeassistant.util.yaml.loader import JSON_TYPE
 
 from . import (
-    area_registry,
     config_validation as cv,
     device_registry,
     entity_registry,
-    floor_registry,
-    label_registry,
     template,
     translation,
 )
-from .group import expand_entity_ids
-from .selector import TargetSelector
+from .selector import (
+    SelectedEntities,
+    TargetSelector,
+    TargetSelectorData,
+    async_extract_referenced_entity_ids,
+)
```

Comment:
We can probably rely on the existing deprecation system for handling most of these imports.